pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -257,6 +257,8 @@ in buildPythonPackage rec {
     ])
   ];
   postInstall = ''
+    find "$out/${python.sitePackages}/torch/include" "$out/${python.sitePackages}/torch/lib" -type f -exec remove-references-to -t ${stdenv.cc} '{}' +
```

Comment:
This is to avoid pulling in the `gcc` compiler executable.

`libgcc_s` isn't affected:
```
$ ldd /nix/store/2zxcx4d2xdyxldhlkv827kxh3fibj9w1-python3.9-pytorch-1.9.0/lib/python3.9/site-packages/torch/lib/libtorch.so  | grep libgcc
	libgcc_s.so.1 => /nix/store/2zchy9mdx3kk166i98cx5sriq2c1bdsn-glibc-2.33-55/lib/libgcc_s.so.1 (0x00007f61403c6000)
```